master/jobmaster/jobmanager.go

```diff
@@ -17,7 +18,7 @@ import (
 // JobManager manages all the job masters, and notify the offline executor to them.
 type JobManager struct {
 	mu         sync.Mutex
-	jobMasters map[model.JobID]JobMaster
+	jobMasters map[model.JobID]system.JobMaster
 
 	idAllocater    *autoid.Allocator
 	resourceMgr    cluster.ResourceMgr
```

Comment:
job manager  should not hold the resource manger

Reply:
yes, I will refactor it in another PR.